docs/source/workshops/bioc-2015.rst

```diff
@@ -45,114 +32,87 @@ Install gcloud
 Configure gcloud compute
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. container:: toggle
-
-    .. container:: header
-
-        There are a few steps to do this.  Click here to **Show/Hide Instructions**
-
-    .. container:: content
-
-      .. include:: /includes/gcloud_compute_configure.rst
+.. include:: /includes/gcloud_compute_configure.rst
 
 Spin up a Google Compute Engine Instance running Bioconductor
```

Comment:
I've been trying to stop myself using the phrase "spin up" in documents and use terminology consistent with commands, APIs, and official Google docs. In this case "create" would be more appropriate.